### PR TITLE
Making suggested changes

### PR DESCRIPTION
**Pull request by vivek-arte**
This PR adds changes suggested by ECC for merging the ZSA changes into the zcash-test-vectors repository

- It changes the comment on the addition of the BIP 340 reference to mention the specific commit the code has been copied from.
- It adds back the generation of orchard_key_components, so that keys can be generated for orchard irrespective.

### CODE REVIEW

File: zcash_test_vectors/orchard/key_components.py [hunk 1]
```diff
@@ -64,15 +63,15 @@ def __init__(self, chaincode, data):
...
+        I   = digest.digest()
         I_L = I[:32]
         I_R = I[32:]
         return cls(I_R, I_L)
 
     def child(self, i):
         assert 0x80000000 <= i and i <= 0xFFFFFFFF
 
-        I = prf_expand(self.chaincode, b'\x81' + self.data + i2leosp(32, i))
+        I   = prf_expand(self.chaincode, b'\x81' + self.data + i2leosp(32, i))
```
Comment (PaulLaux) [inline zcash_test_vectors/orchard/key_components.py:74]: Is this spacing method is compatible with the rest of the package?

File: zcash_test_vectors/orchard_zsa/key_components.py [hunk 2]
```diff
@@ -46,50 +47,42 @@ def randbytes(l):
 
     test_vectors = []
     for i in range(0, 10):
-        sk = SpendingKey(rand.b(32))
+        ki = KeyInit(rand)
```
Comment (PaulLaux) [inline zcash_test_vectors/orchard_zsa/key_components.py:50]: `ki` sounds like a typo of `key`, let's do `k` instead.

**Review by PaulLaux — APPROVED** [review]
minor change is required

File: zcash_test_vectors/orchard/key_components.py (hunk 1 above)
Comment (vivek-arte) [reply zcash_test_vectors/orchard/key_components.py:74]: This is how the original orchard file from upstream spaces it.

File: zcash_test_vectors/orchard_zsa/key_components.py (hunk 2 above)
Comment (vivek-arte) [reply zcash_test_vectors/orchard_zsa/key_components.py]: Done.

**Review by PaulLaux — APPROVED** [review]

**Pull request merged**